wkhtmltopdf/utils.py

```diff
@@ -109,7 +109,7 @@ def wkhtmltopdf(pages, output=None, **kwargs):
 
     return check_output(ck_args, **ck_kwargs)
 
-def convert_to_pdf(filename, header_filename=None, footer_filename=None, cmd_options=None):
+def convert_to_pdf(filename_list, header_filename=None, footer_filename=None, cmd_options=None):
```

Comment:
Excellent idea.

I was debating with myself if I should do that, and whether or not I should keep the name `filename` or change it to `filename_list`. Anyone who calls convert_to_pdf directly could have broken compatibility. 

But then I was reading about the importance of naming, and thought to change the name to `filename_list`.

Either way works for me. Should I do both of the following?
1. Change the name back to 'filename'
2. Add the list coercion.